LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js

```diff
@@ -1172,6 +1172,12 @@ const gCSSProperties2 = {
       { type: 'discrete', options: [ [ 'auto', 'smooth' ] ] }
     ]
   },
+  'scrollbar-width': {
+    // https://w3c.github.io/csswg-drafts/css-scrollbars/#propdef-scrollbar-width
+    types: [
+      { type: 'discrete', options: [ [ 'auto', 'thin', 'none' ] ] }
```

Comment:
I think you can only provide pairs of values here, so if you want to try all three, you'd have to specify the three combinations.

Reply:
Just noticed this issue locally have changed to to just be thin and auto.

Reply:
I think it's possible you may have a bug with `auto` values per the bot results I saw for the WPT `web-animations` tests. You should probably still test `auto` with other values and see what gives.